lib/flutter_map_tappable_polyline.dart

```diff
@@ -56,8 +67,13 @@ class TaggedPolyline extends Polyline {
 }
 
 class TappablePolylineLayer extends StatelessWidget {
+  /// The options allowing tappable polyline tweaks
   final TappablePolylineLayerOptions polylineOpts;
+
+  /// The flutter_map [MapState]
   final MapState map;
+
+  // what is that stream for?
```

Comment:
Can't remember, I'll check

Reply:
Stream used by `flutter_map` to notify us a redraw is required

See `StreamBuilder` in `_build` method.

Reply:
Done.